src/lib/components/configIpModal.svelte

```diff
@@ -64,14 +64,16 @@
         <form onsubmit={handleConfirm}>
             <article class="pb-4">
                 <p>
-                    Gebe die IP-Adresse deiner Steckdose ein, um dich mit ihr zu verbinden.<br />
-                    <span class="opacity-60"
-                        >Die Steckdose muss sich im gleichen Netzwerk befinden. Du kannst die
+                    <span class="text-lg">
+                        Gib die IP-Adresse deiner Steckdose ein, um dich mit ihr zu verbinden.
```

Comment:
Finde ich sonst schon sehr ugs.
```suggestion
                        Gib' die IP-Adresse deiner Steckdose ein, um dich mit ihr zu verbinden.
```